src/js/components/Pagination/README.md

```diff
@@ -196,71 +197,349 @@ nav
 ```
 ## Theme
   
-**pagination**
+**pagination.button.active.background.color**
 
-The possible sizes of the Pagination in terms of its 
-      max-width, font-size and line-height. Expects `object`.
+Background color when the button is active. Expects `string | { dark: string, light: string }`.
 
 Defaults to
 
 ```
-{
-      small: {
-        size: '14px',
-        height: '20px',
-        maxWidth: '336px',
-       },
-      medium: {
-        size: '18px',
-        height: '24px',
-        maxWidth: '432px',
-      },
-      large: {
-        size: '22px',
-        height: '28px',
-        maxWidth: '528px',
-      },
-      xlarge: {
-        size: '26px',
-        height: '32px',
-        maxWidth: '624px',
-      },
-      xxlarge: {
-        size: '34px',
-        height: '40px',
-        maxWidth: '816px',
-      },
-    }
-```
-
-**pagination.extend**
-
-Any additional style for the Pagination. Expects `string | (props) => {}`.
+active-background
+```
+
+**pagination.button.color**
+
+The color of the text label. Expects `string | { dark: string, light: string }`.
+
+Defaults to
+
+```
+text-strong
+```
+
+**pagination.button.hover.background.color**
+
+Background color when the button when hovered. Expects `string | { dark: string, light: string }`.
+
+Defaults to
+
+```
+background-contrast
+```
+
+**pagination.button.hover.color**
+
+The color of the text label when hovered. Expects `string | { dark: string, light: string }`.
 
 Defaults to
 
 ```
 undefined
 ```
 
-**global.edgeSize**
+**pagination.container**
 
-The possible sizes for margin. Expects `object`.
+Any valid Box property affecting the Box 
+    wrapping the pagination controls. Expects `object`.
 
 Defaults to
 
 ```
-{
-    edgeSize: {
-      none: '0px',
-      hair: '1px',
-      xxsmall: '3px',
-      xsmall: '6px',
-      small: '12px',
-      medium: '24px',
-      large: '48px',
-      xlarge: '96px',
-      responsiveBreakpoint: 'small',
-    },
-  }
+undefined
+```
+
+**pagination.container.extend**
+
+Any additional style for the Box wrapping 
+    the pagination controls. Expects `string | (props) => {}`.
+
+Defaults to
+
+```
+undefined
+```
+
+**pagination.control.extend**
+
+Any additional style for each pagination control. Expects `string | (props) => {}`.
```

Comment:
```suggestion
Any additional style for each control. Expects `string | (props) => {}`.
```
Seems that you leaned towards not using `pagination` later on

Reply:
Done